presto-native-execution/scripts/dockerfiles/centos-9-stream-dependency.dockerfile

```diff
@@ -10,11 +10,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM quay.io/centos/centos:stream8
+FROM quay.io/centos/centos:stream9
```

Comment:
Maybe I should rename the file to centos-dependency.dockerfile to remove the version? This would match a similar change for the dockerfile in Velox from `centos-8-stream.dockerfile` to `centos.dockerfile`.

Reply:
Let's do this.